src/Orchard.Web/Modules/Orchard.Widgets/Services/ILayerEvaluationService.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System.Collections.Generic;
+
+namespace Orchard.Widgets.Services
```

Comment:
Hmmmm. None of the other interfaces in the `Orchard.Widgets` module contain their default implementation in the same file.

Plus I always like the idea of a single file containing a single class/ interface.